modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-deploying-the-numa-aware-scheduler_{context}"]
+= Deploying the NUMA-aware secondary pod scheduler
+
+After you have installed the NUMA Resources Operator, configure the pod admittance policy for the required machine profile, create the required machine config pool, and deploy the NUMA-aware secondary scheduler. These steps are described in the following procedure.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+. Create the `KubeletConfig` custom resource that configures the pod admittance policy for the machine profile:
+
+.. Save the following YAML in the `nro-kubeletconfig.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: cnf-worker-tuning
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      cnf-worker-tuning: enabled
+  kubeletConfig:
+    cpuManagerPolicy: "static"
+    cpuManagerReconcilePeriod: "5s"
+    reservedSystemCPUs: "0,1"
+    memoryManagerPolicy: "Static"
+    evictionHard:
+      memory.available: "100Mi"
+    kubeReserved:
+      memory: "512Mi"
+    reservedMemory:
+      - numaNode: 0
+  limits:
+    memory: "1124Mi"
+    systemReserved:
+      memory: "512Mi"
+    topologyManagerPolicy: "single-numa-node"
+----
```

Comment:
yes please. It should read like
```
...
topologyManagerPolicy: "single-numa-node"
topologyManagerScope: "pod"
```

this will be removed when the known limitations are addressed, later in the cycle